workflow/schemas/config.schema.yaml

```diff
@@ -147,6 +147,17 @@ properties:
         type: string
         description: name or path to docker/singularity container
 
+  reference:
+    type: object
+    description: reference used by alignment
```

Comment:
It doesn't make sense to say `reference used for alignment` when the subcategories are artifacts and background. Maybe simply `collection of reference files` or something similar